cli/printer/text.go

```diff
@@ -74,7 +74,11 @@ func (t *TextPrinter) PrintDeploySummaries(deploys ...*models.DeploySummary) err
 
 func (t *TextPrinter) PrintEnvironments(environments ...*models.Environment) error {
 	getScale := func(e *models.Environment) string {
-		return fmt.Sprintf("%d:%d:%d", e.MinScale, e.CurrentScale, e.MaxScale)
+		if e.EnvironmentType != models.EnvironmentTypeDynamic {
+			return fmt.Sprintf("%d/%d", e.CurrentScale, e.DesiredScale)
+		}
+
+		return "n/a"
```

Comment:
[nit] would displaying a `-` look better?

Reply:
Don't feel strongly either way. Will change it to `-`. ✅